Kitodo/src/main/java/org/kitodo/production/forms/TemplateForm.java

```diff
@@ -37,6 +37,7 @@
 import org.kitodo.production.helper.SelectItemList;
 import org.kitodo.production.model.LazyDTOModel;
 import org.kitodo.production.services.ServiceManager;
+import org.kitodo.production.services.data.TemplateService;
```

Comment:
`TemplateService` wasn't imported in `TemplateForm` until now? That should have made us suspicous!